turbo/templates/turbo/turbo_stream_source.html

```diff
@@ -0,0 +1,2 @@
+<turbo-channels-stream-source signed-channel-name="{{ signed_channel_name }}">
+</turbo-channels-stream-source>
```

Comment:
```suggestion
<turbo-channels-stream-source signed-channel-name="{{ signed_channel_name }}" />
```
Small detail, as the tag does not have a content, we can use a self-closing tag